app/src/cli/commands/help.ts

```diff
@@ -0,0 +1,80 @@
+import * as chalk from 'chalk'
+
+import { commands, ICommandModule, IOption } from '../load-commands'
+
+import { $0, dasherizeOption, printTable } from '../util'
+
+const command: ICommandModule = {
+  command: 'help [command]',
+  description: 'HALP!!1',
```

Comment:
Let's keep this classy and say something like:

> Show the help page for a command

Reply:
✅